src/itemRadioGroup/index.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { PureComponent, cloneElement } from 'react'
+
+import { ItemRadioStatus, ItemRadioProps } from 'itemRadio'
+import ItemsList from 'itemsList'
+
+export interface ItemRadioGroupProps {
+  readonly name: string
+  readonly children: React.ReactElement<ItemRadioProps>[]
+  readonly className?: Classcat.Class
+  readonly value?: string | number | boolean
+  readonly onChange?: (obj: OnChangeParameters) => void
+  readonly status?: ItemRadioStatus
+  readonly withSeparators?: boolean
+  readonly withChevrons?: boolean
+}
+
+interface ItemRadioGroupState {
+  value: string | number | boolean
+}
+
+class ItemRadioGroup extends PureComponent<ItemRadioGroupProps, ItemRadioGroupState> {
+  static defaultProps: Partial<ItemRadioGroupProps> = {
+    value: '',
+    className: '',
+    onChange() {},
+    withSeparators: false,
+    withChevrons: false,
+  }
+
+  state: ItemRadioGroupState = {
+    value: this.props.value,
+  }
+
+  onChange = ({ name, value }: OnChangeParameters) => {
+    this.props.onChange({ name, value })
+    this.setState({ value })
+  }
+
+  render() {
+    const { children, status, name, className, withSeparators, withChevrons } = this.props
+    return (
+      <ItemsList withSeparators={withSeparators} className={className}>
```

Comment:
Would be good to add a `role="radiogroup"`, and maybe I'll add a ticket to check on keyboard support
https://www.w3.org/TR/2017/WD-wai-aria-practices-1.1-20170628/examples/radio/radio-1/radio-1.html

Reply:
Added the role attribute ;)